docs/src/content/hardhat-network/docs/explanation/mining-modes.md

```diff
@@ -47,11 +47,11 @@ networks: {
 
 ## Removing and replacing transactions
 
-Transactions in the mempool can be removed using the `hardhat_dropTransaction` method:
+Transactions in the mempool can be removed using the [`dropTransaction`](</hardhat-network-helpers/docs/reference#droptransaction(txhash)>) network helper:
 
 ```js
 const txHash = "0xabc...";
-await network.provider.send("hardhat_dropTransaction", [txHash]);
+await helpers.dropTransaction(txHash);
```

Comment:
`### Using RPC methods` still uses the provider.

Reply:
Yes, we don't have helpers to change the mining modes yet. [HH-921](https://linear.app/nomic-foundation/issue/HH-921/add-network-helper-to-configure-mining-behavior) tracks that.